.github/workflows/build.yml

```
@@ -0,0 +1,35 @@
+---
+# This workflow will build a Java project with Maven, and cache/restore any dependencies to improve the workflow execution time
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
+
+name: Java CI with Maven
+
+on:
+  pull_request:
+    branches: [ main ]
+  workflow_dispatch:
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f # v2
+    - name: Set up JDK 8
+      uses: actions/setup-java@8764a52df183aa0ccea74521dfd9d506ffc7a19a # v2
+      with:
+        java-version: '8'
+        distribution: 'adopt'
+        cache: maven
+    - name: Build with Maven
+      run: mvn package
```

I think it would be good if [the ~ 12k lines of output](https://github.com/UCLALibrary/laptop_availability/runs/5502865361?check_suite_focus=true#step:4:12027) could be reduced:
```suggestion
      run: mvn -ntp package
```